modules/TargetInfo.lua

```diff
@@ -12,6 +12,10 @@ local internal = "internal"
 
 local ValidAnchors = { "TOPLEFT", "TOPRIGHT", "BOTTOMLEFT", "BOTTOMRIGHT", "CENTER" }
 
+---- Fulzamoth - 2019-09-04 : support for cooldowns on target buffs/debuffs (classic)
+local LibClassicDurations = LibStub("LibClassicDurations")
```

Comment:
Add the "true" second argument to LibStub so this won't fail if the addon isn't loaded, please.